### ISSUE TITLE
Download code error

### ISSUE THREAD

**Issue by Brocoleta** [FlutterFlow/flutterflow-issues#3264]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When i download the project and i try to build ios, it says:

Expected ios/Runner.xcodeproj/project.pbxproj but this file is missing.
Application not configured for iOS

### Expected Behavior

It must build wth no errors

### Steps to Reproduce

1. Download code
2. Run flutter build ios

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

-

### Visual documentation

-

### Environment

```markdown
- FlutterFlow version: 
- Platform:
- Browser name and version:
- Operating system and version affected:
```

**Comment by rzambroni** [comment]
Hello @Brocoleta, we will need you to file a complete issue, there is not much we can do with the information provided on this ticket.

**Comment by Brocoleta** [comment]
im using:

FlutterFlow v4.1.64 released June 27, 2024
Flutter version is 3.22.2

i cannot give you the error code, because the error is when i download the code and try tu build ios, also this error happens when i try to open Runner.xcodeproj:

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/48292258/53f8f4ae-42f6-42cc-be54-32a6654e732b)

**Comment by rzambroni** [comment]
We will need the bug report anyway to at least see if your code downloads correctly.

here is a guide on how to get it: [https://docs.flutterflow.io/troubleshooting/submit-bug-reports](https://docs.flutterflow.io/troubleshooting/submit-bug-reports)

**Comment by Brocoleta** [comment]
i could only download it for a specific widget,(its not related to the problem):

ITEek/Lqw4xJpbxI0Kr2d8JsgQMwJ0QlUIInj+0bWRMhfJjmPJoLO8/sSFtLYtfnS21beWeYing1+vDHuIX9FfEVNQqaGoc6yYRXQRbMQlujRr6yCZeCVGtBHeRmGFC74abQkR4lCM1qSCxg7TiqBvCNG3qCf9qOYwx5e6fDbOY=

**Comment by rzambroni** [comment]
That's fine, it is just so we have access to your project otherwise we are blind.

**Comment by msusviela** [comment]
Hi @Brocoleta. Sorry for the late response. I could download the code with no errors. Yesterday a new version of FlutterFlow was released (4.1.75). Is this still an issue for you in the new version?

**Comment by msusviela** [comment]
Closing this issue since we haven't heard back from you. If you are still facing the problem please submit a new issue. Have a great day!
